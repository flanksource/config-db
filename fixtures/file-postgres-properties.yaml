apiVersion: configs.flanksource.com/v1
kind: ScrapeConfig
metadata:
  name: file-postgres-properties-scraper
spec:
  file:
    - format: properties
      type: PostgreSQLProperties
      id: postgresql-properties
      paths:
        - ./fixtures/file-postgres-properties.conf
