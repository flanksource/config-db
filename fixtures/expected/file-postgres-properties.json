[
  {
    "last_modified": "0001-01-01T00:00:00Z",
    "id": "postgresql-properties",
    "config_class": "PostgreSQLProperties",
    "config_type": "PostgreSQLProperties",
    "config": {
      "authentication_timeout": "1min",
      "checkpoint_completion_target": "0.5",
      "checkpoint_flush_after": "256kB",
      "cluster_name": "'main'",
      "cron.database_name": "'postgres'",
      "data_directory": "'/var/lib/postgresql/data'",
      "db_user_namespace": "off",
      "default_text_search_config": "'pg_catalog.english'",
      "effective_cache_size": "128MB",
      "extra_float_digits": "0",
      "hba_file": "'/etc/postgresql/pg_hba.conf'",
      "ident_file": "'/etc/postgresql/pg_ident.conf'",
      "jit_provider": "'llvmjit'",
      "lc_messages": "'en_US.UTF-8'",
      "lc_monetary": "'en_US.UTF-8'",
      "lc_numeric": "'en_US.UTF-8'",
      "lc_time": "'en_US.UTF-8'",
      "listen_addresses": "'*'",
      "log_destination": "'csvlog'",
      "log_directory": "'/var/log/postgresql'",
      "log_file_mode": "0640",
      "log_filename": "'postgresql.log'",
      "log_line_prefix": "'%h %m [%p] %q%u@%d '",
      "log_rotation_age": "0",
      "log_rotation_size": "0",
      "log_statement": "'all'",
      "log_timezone": "'UTC'",
      "logging_collector": "on",
      "max_replication_slots": "5",
      "max_slot_wal_keep_size": "1024",
      "max_wal_senders": "10",
      "password_encryption": "scram-sha-256",
      "pgsodium.getkey_script": "'/usr/lib/postgresql/14/bin/pgsodium_getkey_urandom.sh'",
      "pljava.libjvm_location": "'/usr/lib/jvm/java-11-openjdk-amd64/lib/server/libjvm.so'",
      "row_security": "on",
      "shared_buffers": "128MB",
      "shared_preload_libraries": "'pg_stat_statements, pgaudit, plpgsql, plpgsql_check, pg_cron, pg_net, pgsodium'",
      "ssl": "off",
      "ssl_ca_file": "''",
      "ssl_cert_file": "''",
      "ssl_ciphers": "'HIGH:MEDIUM:+3DES:!aNULL'",
      "ssl_crl_dir": "''",
      "ssl_crl_file": "''",
      "ssl_dh_params_file": "''",
      "ssl_ecdh_curve": "'prime256v1'",
      "ssl_key_file": "''",
      "ssl_max_protocol_version": "''",
      "ssl_min_protocol_version": "'TLSv1.2'",
      "ssl_passphrase_command": "''",
      "ssl_passphrase_command_supports_reload": "off",
      "ssl_prefer_server_ciphers": "on",
      "timezone": "'UTC'",
      "unix_socket_directories": "'/var/run/postgresql'",
      "wal_level": "logical"
    }
  }
]
